### PR TITLE
Update SafeQueue link to a maintained version

### PR DESCRIPTION
**Pull request by dpslwk**

### CODE REVIEW

**Comment by eigan** [comment]
I could merge this, but it probably won't help. I have no control over the website. 

ping @josenicomaia

File: resources/views/partials/main-nav.blade.php [hunk 1]
```diff
@@ -15,6 +15,6 @@
         <h6 class="dropdown-header">Community packages</h6>
         <a class="dropdown-item" target="_blank" href="https://github.com/dave-redfern/laravel-doctrine-behaviours">Laravel Doctrine Behaviours</a>
         <a class="dropdown-item" target="_blank" href="https://github.com/bharley/laraveldoctrine-hashids">Hashids</a>
-        <a class="dropdown-item" target="_blank" href="https://github.com/maxbrokman/SafeQueue">SafeQueue</a>
+        <a class="dropdown-item" target="_blank" href="https://github.com/digband/safe-queue">SafeQueue</a>
```
Comment (josenicomaia) [inline resources/views/partials/main-nav.blade.php]: I am getting 404 for this page. Is it broken?
`https://github.com/digband/safe-queue`

**Review by josenicomaia — CHANGES_REQUESTED** [review]

**Comment by josenicomaia** [comment]
@eigan No problem. I am follow this thread.

**Comment by dpslwk** [comment]
sorry fixed the typo

do wonder if we may end up having to fork and take over the repo under the laravel-doctrine name at some point in the future

**Review by josenicomaia — APPROVED** [review]

**Pull request merged**